packages/flow-remove-types/index.js

```diff
@@ -397,6 +407,7 @@ function getTrailingLineNode(context, node) {
 // Creates a zero-width "node" with a value to splice at that position.
 // WARNING: This is only safe to use when source maps are off!
 function getSpliceNodeAtPos(context, pos, loc, value) {
+  context.bytesAdded += value.length;
```

Comment:
This is _technically_ mixing JS string length (UTF-16 code units) and UTF-8 bytes. Not a problem yet because the only call to `getSpliceNodeAtPos` uses a plain ASCII string (`' =>'`), but maybe we should future-proof this?